README.md

```diff
@@ -14,121 +14,383 @@ A Deno client for the [NATS messaging system](https://nats.io).
 ```bash
 import * as nats from 'https://raw.githubusercontent.com/nats-io/nats.deno/master/src/mod.ts'
 ```
-
-# Basic Usage
-
 The Deno client is under active development. All tests are passing, but the APIs will change
 slightly - See the [TODO](TODO.md).
 
-```javascript
+
+
+## Basics
+
+
+### Connecting to a nats-server
+
+To connect to a server you use the `connect()` function. It returns
+a connection that you can use to interact with the server.
+
+By default, a connection will attempt to auto-reconnect when dropped
+due to some networking type error. Messages that have not been
+sent to the server when a disconnect happens are lost. A client can queue
+new messages to be sent when the connection resumes. If the connection
+cannot be re-established the client will give up and `close` the connection.
+
+To learn when a connection closes, wait for the promise returned by the `status()`
+function. If the close was due to an error, it will provide resolve to an error.
```

Comment:
typo: ", it will resolve to an error" instead?

Reply:
fixed